api/admin/connecthandler/admin_connect.go

```diff
@@ -0,0 +1,305 @@
+) (*connect.Response[emptypb.Empty], error) {
+	if err := s.Admin.MemoryProfile(nil, nil, &api.EmptyReply{}); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+	return connect.NewResponse(&emptypb.Empty{}), nil
+}
+
+// LockProfile runs a lock profile writing to the specified file
+func (s *ConnectAdminService) LockProfile(
+	_ context.Context,
+	_ *connect.Request[emptypb.Empty],
+) (*connect.Response[emptypb.Empty], error) {
+	if err := s.Admin.LockProfile(nil, nil, &api.EmptyReply{}); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+	return connect.NewResponse(&emptypb.Empty{}), nil
+}
+
+// Alias attempts to alias an HTTP endpoint to a new name
+func (s *ConnectAdminService) Alias(
+	_ context.Context,
+	req *connect.Request[adminv1.AliasArgs],
+) (*connect.Response[emptypb.Empty], error) {
+	jsonArgs := &admin.AliasArgs{
+		Endpoint: req.Msg.Endpoint,
+		Alias:    req.Msg.Alias,
+	}
+
+	if err := s.Admin.Alias(nil, jsonArgs, &api.EmptyReply{}); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+
+	return connect.NewResponse(&emptypb.Empty{}), nil
+}
+
+// AliasChain attempts to alias a chain to a new name
+func (s *ConnectAdminService) AliasChain(
+	_ context.Context,
+	req *connect.Request[adminv1.AliasChainArgs],
+) (*connect.Response[emptypb.Empty], error) {
+	jsonArgs := &admin.AliasChainArgs{
+		Chain: req.Msg.Chain,
+		Alias: req.Msg.Alias,
+	}
+
+	if err := s.Admin.AliasChain(nil, jsonArgs, &api.EmptyReply{}); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+
+	return connect.NewResponse(&emptypb.Empty{}), nil
+}
+
+// GetChainAliases returns the aliases of the chain
+func (s *ConnectAdminService) GetChainAliases(
+	_ context.Context,
+	req *connect.Request[adminv1.GetChainAliasesArgs],
+) (*connect.Response[adminv1.GetChainAliasesReply], error) {
+	jsonArgs := &admin.GetChainAliasesArgs{
+		Chain: req.Msg.Chain,
+	}
+
+	var jsonReply admin.GetChainAliasesReply
+	if err := s.Admin.GetChainAliases(nil, jsonArgs, &jsonReply); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+
+	reply := &adminv1.GetChainAliasesReply{
+		Aliases: jsonReply.Aliases,
+	}
+
+	return connect.NewResponse(reply), nil
+}
+
+// Stacktrace returns the current global stacktrace
+func (s *ConnectAdminService) Stacktrace(
+	_ context.Context,
+	_ *connect.Request[emptypb.Empty],
+) (*connect.Response[emptypb.Empty], error) {
+	if err := s.Admin.Stacktrace(nil, nil, &api.EmptyReply{}); err != nil {
+		return nil, connect.NewError(connect.CodeInternal, err)
+	}
+	return connect.NewResponse(&emptypb.Empty{}), nil
+}
+
+// SetLoggerLevel sets the log level and/or display level for loggers
+func (s *ConnectAdminService) SetLoggerLevel(
+	_ context.Context,
+	req *connect.Request[adminv1.SetLoggerLevelArgs],
+) (*connect.Response[adminv1.SetLoggerLevelReply], error) {
+	// Convert the request to the format used by the JSON-RPC API
```

Comment:
This is overly verbose, the code here is self-documenting so we can avoid the comment here